using-openspace/ui-panels/record/index.md

```diff
@@ -144,7 +159,12 @@ The `Output Frames` option determines whether screen shots are generated for the
 The resulting images may be imported into a program that can render them into a movie file.
 
 
+### Options: Hide User Interface Components on Playback
+The `Hide GUI on playback` and `Hide dashboards on playback` options can be used to hide the user interface overlays while a recording is being played back. This can for example be useful when outputting the frames of a recording, to not show the user interface in the outputed frames. The overlays will be shown again once the playback is finished.
```

Comment:
```suggestion
The {menuselection}`Hide GUI on playback` and {menuselection}`Hide dashboards on playback` options can be used to hide the user interface overlays while a recording is being played back. This can, for example, be useful when outputting the frames of a recording to not show the user interface in the outputted frames. The overlays will be shown again once the playback is finished.
```

?